static/js/apps/explore/app.tsx

```diff
@@ -214,6 +216,13 @@ export function App(props: { isDemo: boolean }): JSX.Element {
       pageMetadata.pageConfig &&
       pageMetadata.pageConfig.categories
     ) {
+      // If the response is a single PLACE_OVERVIEW tile, redirect to the place explorer.
+      if (isPlaceOverviewOnly(pageMetadata)) {
+        console.log('Redirecting to place explorer');
+        let url = 'place/';
```

Comment:
Add a leading slash to the url. Otherwise, if i start from `http://localhost:8080/explore/` (with the trailing slash), the redirect takes me to `/explore/place/<dcid>`.

Reply:
hm interesting, im not seeing that happening when I run local though. Still done.

Reply:
Hm could be because im running in the `./run_cdc_dev_docker.sh` setup